point_viewer_grpc/src/service.rs

```diff
@@ -213,7 +223,7 @@ impl proto_grpc::Octree for OctreeService {
         let frustum_matrix = projection_matrix.concat(&view_transform.into());
         stream_points_back_to_sink(
             OctreeQuery::Frustum(frustum_matrix),
-            self.octree.clone(),
+            Arc::clone(&self.get_service_data(&req.octree_id)),
```

Comment:
Do we need to `Arc::clone` here?

Reply:
Good catch, this was some leftover...
Done!